pkg/skaffold/parser/config.go

```
@@ -313,3 +314,16 @@ func unmatchedProfiles(activatedProfiles map[string]string, allProfiles []string
 	}
 	return unmatched
 }
+
+func makeConfigPathAbsolute(dependentConfigPath string, currentConfigPath string) string {
+	if dependentConfigPath == "" || filepath.IsAbs(dependentConfigPath) || util.IsURL(dependentConfigPath) {
+		return dependentConfigPath
+	}
+	// if current config is either a URL or STDIN, base path should be CWD
+	if util.IsURL(currentConfigPath) || currentConfigPath == "-" {
+		cwd, _ := util.RealWorkDir()
```

maybe add a comment on why we can safely ignore the error here?